infra/base-images/base-runner/bad_build_check

```diff
@@ -81,7 +82,7 @@ function check_instrumentation {
     AFL_NO_UI=1 SKIP_SEED_CORPUS=1 timeout --preserve-status -s INT 20s run_fuzzer $FUZZER_NAME &>$FUZZER_OUTPUT
     CHECK_PASSED=$(egrep "All set and ready to roll" -c $FUZZER_OUTPUT)
     if (( $CHECK_PASSED == 0 )); then
-      echo "BAD BUILD: $FUZZER does not seem to have coverage instrumentation."
+      echo "BAD BUILD: fuzzing $FUZZER with afl-fuzz failed."
```

Comment:
Is there a more concrete reason why fuzzing with afl-fuzz would fail here? e.g. in libFuzzer case it's clearly instrumentation issue. Having a more distinguishable explanation here might be helpful.

Reply:
There are a couple reasons including instrumentation being broken, afl-fuzz not being present, and many others I'm sure. Rather than enumerate all of them I think it makes most sense to surface any failure and let the user determine which from the output.